README.md

```diff
@@ -5,4 +5,4 @@ I've been using git a long time and I still love it!
 
 I especially love pull requests and issue tracking, it really makes collaboration easier.
 
-Please see the [LICENSE](LICENSE) file.
+Please see the [LICENSE](LICENSE) file for details about what you are allowed to do with code from this project.
```

Comment:
This looks OK, but maybe we should review our license choice! I hear the MIT license is very popular these days.